ISISComputingGroup/JSON_bourne
# Ticket2239: Make dataweb look like GUI

see https://github.com/ISISComputingGroup/IBEX/issues/2239

**To Test**
- Turn on or off instruments (e.g. demo) and see if the status and colour changes
- Connect with different users and see if users changes
- See if Inst. Time and Run Time change over time as expected
